src/database.rs

```diff
@@ -494,6 +521,7 @@ impl Database {
                 room_id TEXT NOT NULL,
                 content_value TEXT NOT NULL,
                 type TEXT NOT NULL,
+                msgtype TEXT,
```

Comment:
Don't you also need a migration step for anyone with an existing database?

Reply:
I was kinda hoping we can get away without migration logic since nobody uses Seshat yet.

Reply:
Well, we know at least @ara4n does... More people may be starting to try it... Where do you draw the line to say "now we need migration"? Also, how does it fail when not recreated with the new schema? Some kind of silent explosion...?

Reply:
I would assume a 1.0 release means just that, from here on stable database migrations unless a  new major release happens (e.g. 2.0). If you feel like we should spend time on migrations, sure will do so.

Opening the database would result in an error just like with normal data corruption.

Reply:
My main concern is around how it manifests for Riot users. What does "results in an error" mean for the Riot users? A console error they will never look at? Something else?

At the moment, Seshat is basically silent in Riot, so there would be no way to know if something like this has gone wrong. I think we either need migration here so that it silently continues to work for all users AND / OR some bit of UI in Riot to surface the problem for user action.

If we left this as a silent failure for the Riot user, then it just adds to a sense that Riot is flaky and drives people away.

Maybe it's best to chat about this in the room and get feedback from anyone already using it.

Reply:
From room history, we're taking on a higher risk tolerance here than I imagined. If your plan is no commitment to stability until 1.0, then I suppose it can't exit labs in Riot until after Seshat 1.0 and stability is assured.

Reply:
I would think once we start shipping compiled modules Seshat should be stabilized, even if we leave it behind labs. If it's just behind a labs flag but shipped many people will want to try it out. 

Since we're still in "compile it on your own" mode making breaking changes seems OK to me.